WooCommerce/UITestsFoundation/Screens/Login/PrologueScreen.swift

```diff
@@ -4,7 +4,7 @@ import XCTest
 public final class PrologueScreen: ScreenObject {
 
     private let titleLabelGetter: (XCUIApplication) -> XCUIElement = {
-        $0.staticTexts["prologue-title-label"]
+        $0.staticTexts["The ecommerce platform that grows with you"]
```

Comment:
Using the label title would not be a reliable fix, in case the title is renamed in the future. I'd suggest setting the accessibility ID for the label instead:

<img width="685" alt="image" src="https://github.com/user-attachments/assets/4a22dc83-6e90-407d-8e61-d818b7ea5361" />

That way, we can keep the check for `prologue-title-label` to get the title label.

Reply:
Good point! Done in [`54662ca`](https://github.com/woocommerce/woocommerce-ios/pull/15026/commits/54662cac7b8ca4331fd1722c55a66f4294a925f8). 🙇